plugins/env/env.go

```diff
@@ -13,12 +13,29 @@ import (
 
 type plugin struct {
 	plugins.PluginBase
+	extraVars map[string]string
 }
 
 type payloadType struct {
 	Env map[string]string `json:"env"`
 }
 
+type config struct {
+	Extra map[string]string `json:"extra"`
+}
+
+var configSchema = schematypes.Object{
+	Properties: schematypes.Properties{
+		"extra": schematypes.Map{
```

Comment:
Can't we list these as `static` ie. static environment variables... Or `constant`